e2e/cypress/integration/account_settings/general/nickname_spec.js

```diff
@@ -109,14 +114,84 @@ describe('Account Settings > Sidebar > General', () => {
         cy.get('#teamMembersModal').should('be.visible').within(() => cy.get('.close').click());
     });
 
-    it('Clear the nickname', () => {
+    it('MM-T2060 Nickname and username styles', () => {
+        cy.apiAdminLogin();
+        cy.apiCreateChannel(testTeam.id, 'channel-test', 'Channel').then(({channel}) => {
+            // # Go to test channel
+            cy.visit(`/${testTeam.name}/channels/${channel.name}`);
+
+            // # Click 'Add Members'
+            cy.get('#channelHeaderTitle').click();
+            cy.get('#channelAddMembers').click();
+
+            // * Verify that the username span contains the '@' symbol and the dark colour
+            cy.get('#multiSelectList > div > .more-modal__details > .more-modal__name > span').should('contain', '@').and('have.css', 'color', 'rgb(61, 60, 64)');
+
+            // # Close modal
+            cy.get('body').type('{esc}');
+
+            // # Go to manage members modal
+            cy.get('#channelMemberIcon').click();
+            cy.get('#member-list-popover').should('be.visible').within(() => {
+                cy.findByTestId('membersModal').click();
+            });
+
+            // * Verify that the username span contains the '@' symbol and the dark colour
+            cy.get('.more-modal__details > .more-modal__name').should('contain', '@').and('have.css', 'color', 'rgb(61, 60, 64)');
```

Comment:
shouldn't we check for a class to exist instead of specific css values?

Reply:
from what I understood in the ticket is to verify the colour 🤷 also it would be the case if we used a unified class for those elements and have a clean css altogether but at the moment it's not the case :/